docs/ide-support.mdx

```diff
@@ -17,7 +17,16 @@ Melos will create an IntelliJ project module for each package in your Melos
 workspace, and will create flutter and dart Run configurations for your
 `main.dart`s and your package's test suite.
 
-![Generated Run configurations](/assets/intellij-run-configurations.png)
+<img src="/assets/intellij-run-configurations.png" alt="Generated Run configurations" height="600"/>
+
+### `melos.yaml`
+
+Autocompletion and validation of `melos.yaml` can be enabled in IntelliJ by going to `Settings` > `Languages & Frameworks` > `Schemas and DTDs` > `JSON Schema Mappings` and adding the Melos schema:
```

Comment:
Can you wrap this so that the line length is <80?

Reply:
Done 👍